posts/2016-11-02-serverless-v1.1.0.md

```diff
@@ -0,0 +1,44 @@
+---
+title:  New Release: 1.1.0
+description:  Today we're releasing 1.1.0 of the Serverless Framework
+date:  2016-10-27
+thumbnail:  https://cloud.githubusercontent.com/assets/20538501/19769976/25c14630-9c23-11e6-9698-8cda3b8809d8.jpg
+layout:  Post
+---
+
+With the release of 1.0 a few weeks ago we're now settling into our bi-weekly release cycle and can announce our next release today with 1.1.0.
+
+We decided to go with a bi-weekly release cycle to force us to release new features and bug fixes quickly and to really focus on shipping for our community. At the same time we're investing heavily in testing, integration tests, code review and really making sure that our stability is high.
```

Comment:
"heavily in testing, integration tests, code review" -> "heavily in unit tests, integration tests, code review"